spec/views/exchanges/employer_applications/_reinstate_benefit_application_modal.html.erb_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
@nks2109  - as discussed earlier can you remove this & add a todo comment in cuke